etc/motions/stand_up_back.json

```diff
@@ -3,167 +3,44 @@
   "initial_positions": {
     "head": {
       "yaw": 0.0,
-      "pitch": 0.0
+      "pitch": 0.3839724354387525
     },
     "left_arm": {
-      "shoulder_pitch": 1.57,
-      "shoulder_roll": 0.0,
-      "elbow_yaw": 0.0,
-      "elbow_roll": 0.0,
-      "wrist_yaw": 0.0,
-      "hand": 0.0
+      "elbow_roll": -1.2578380107879639,
+      "elbow_yaw": 0.20244598388671875,
+      "hand": 0,
+      "shoulder_pitch": 1.918992042541504,
+      "shoulder_roll": 0.12421202659606934,
+      "wrist_yaw": -0.052197933197021484
     },
     "right_arm": {
-      "shoulder_pitch": 1.57,
-      "shoulder_roll": 0.0,
-      "elbow_yaw": 0.0,
-      "elbow_roll": 0.0,
-      "wrist_yaw": 0.0,
-      "hand": 0.0
+      "elbow_roll": 1.376039981842041,
+      "elbow_yaw": -0.14883995056152344,
+      "hand": 0,
+      "shoulder_pitch": 1.9620280265808105,
+      "shoulder_roll": -0.26695799827575684,
+      "wrist_yaw": 0.056715965270996094
     },
```

Comment:
Why did you change the order?

Reply:
This is directly copied from Twix and is in alphabetical order

Reply:
okay